packages/vm/tests/retesteth/README.md

```diff
@@ -1,16 +1,23 @@
-This folder contains a script to run with `ts-node` together with `retesteth`. 
+This folder contains a script to run with `ts-node` together with `retesteth`.
 
-Instructions
-====
+# Instructions
 
-Clone `ethereum/tests`.
+Clone `ethereum/tests` (either in the `ethereumjs-monorepo` or elsewhere).
 
-Install `retesteth v0.2.2`.
+Configure and run `transition-tool`:
 
-Ensure that in `retesteth/clients/ethereumjs/start.sh` the right path of `transition-tool` is set.
+Note: All paths are relative paths to the `VM` package root.
 
-Now run `retesteth`:
+1. If you change the port number in `transition-cluster.ts` to anything other than 3000 (or run `transition-cluster` on a separate machine or different IP address from retesteth), update `tests/vm/retesteth/clients/ethereumjs/start.sh` to reflect the right IP and port.
 
-`./retesteth -t GeneralStateTests -- --testpath $ETHEREUM_TESTPATH --datadir #RETESTETH_CLIENTS --clients "ethereumjs"`
+2. From VM package root directory, run `npx ts-node tests/retesteth/transition-cluster.ts`
```

Comment:
Why `npx`?

Reply:
Because I don't have ts-node installed globally 😬

Reply:
I don't really mind but I think it is more clean to just do `ts-node` instead of `npx ts-node`.

Reply:
That's totally fine. Won't hurt my feelings if you update it